src/client/program.cpp

```diff
@@ -0,0 +1,50 @@
+#include "program.hpp"
+#include "client.hpp"
+#include "job.hpp"
+#include "job_queue.hpp"
+#include "fail_if.hpp"
+
+#include <thread>
+
+bool JobHandle(Job & t_job){
+    jobQ.push(t_job);
+    jobQ.pop_res(t_job);
+
+    return t_job.bool_value;
+}
```

Comment:
you can't get a response right away like this. it may take a few seconds for a job to be completed and your res queue will be empty.  You either need to loop and check the queue has a new response.